scripts/benchmarks_ci.py

```diff
@@ -342,7 +342,8 @@ def __run_benchview_scripts(args: list, verbose: bool) -> None:
     benchviewpy.machinedata(working_directory=bin_directory)
 
     for framework in args.frameworks:
-        buildinfo = __get_build_info(args, framework)
+        target_framework_moniker = FrameworkAction.get_target_framework_moniker(framework)
+        buildinfo = __get_build_info(args, )
```

Comment:
Missing second argument.

Reply:
good catch! and a noobish bug on my side

Reply:
I recommend using `pyflakes` when modifying the python scripts, and, ideally, open an issue to create a CI that run linters on this code.